Point#251

yiyingl4 opened the pull request:
1. Fix shadow
2. Decrease spaces
3. make it green on hover

ddfridley reviewed (changes requested):
just quick changes.  Thanks.

ddfridley commented on app/components/point.jsx:
```diff
@@ -206,13 +211,13 @@ const useStylesFromThemeFunction = createUseStyles(theme => ({
   sharedBorderStyle: {
     borderRadius: '0.9375rem',
-    boxShadow: theme.boxShadow,
+    boxShadow: theme.none,
```
our maybe it should be 'unset' - please check.

ddfridley reviewed (changes requested):
> [onClick the card should be green-filled as well](https://www.figma.com/design/EFnBPfEEejKi6CVJGAQWJh/Civic-Pursuit-Tool%3A-Wireframe-%2B-Design?node-id=2061-9077&t=qCtf0Kbet0GrsU3A-4)

Means that if the user clicks on the a card, like the Point 1 card, that it should be selected just as if the clicked on the select as lead button.

Please check the shadow around the cards, it seems like there is less shadow on the left and top in figma than what we have right now.

And see comments below.

yiyingl4 commented:
Hello David,

I've fixed the shadow around the cards. Thank you for remind!
I found that the color of the information part in the primary selected is not green, so I fixed it.
For onClick the card, I changed the state from default to selected when the user clicks the card. I have tested it in Primary Default.

Thank you!

ddfridley commented on app/components/point.jsx:
```diff
@@ -206,13 +225,13 @@ const useStylesFromThemeFunction = createUseStyles(theme => ({
   sharedBorderStyle: {
     borderRadius: '0.9375rem',
-    boxShadow: theme.boxShadow,
+    boxShadow: '3px 3px 5px rgba(0, 0, 0, 0.05)',
```
please put this in theme, give it a name like pointShadow and use rem instead of px.

ddfridley reviewed (changes requested):
I'm really really sorry. The comment about changing to green on click was meant for a different issue. Some how I put it in the wrong place.  It's a good implementation, but there are many cases where we don't want Point to turn green when the user click on it.  The comment is for one of the issues that is using the Point component. 

See one other comment too.  Thanks!

yiyingl4 commented:
Hello David,
Thank you for your comment. I created a new theme in theme.js and passed it to point.jsx. Please let me know if there is other thing I can improve. Thank you!

ddfridley reviewed (changes requested):
All the changes look good, but I see that the gap at the bottom, has shrunk - compared to what it was in master.  Can you do something to fix it.

<img width="338" alt="image" src="https://github.com/user-attachments/assets/102351c0-a9f5-453a-a183-6be5f2239ef6" />

this is http://localhost:6006/?path=/story/point--primary-default

With storybook running, you can `git checkout master` to see what's in master and 'git checkout point#251` to switch back.  It's nice to be able to compare this way.

yiyingl4 commented:
Thank you for your comment, David!
Please look at the newest version to see if it is good! Thank you!

ddfridley reviewed (approved):
Looks great. Thanks!
